### PR TITLE
Upgrade inverse-finance to schema 1.3.0

### PR DESCRIPTION
**Pull request by tnkrxyz**
- [x] verify syncing working https://thegraph.com/hosted-service/subgraph/tnkrxyz/inverse-finance?version=pending
- [x] verify information correct

### CODE REVIEW

File: subgraphs/inverse-finance/src/common/helpers.ts [hunk 1]
```diff
@@ -129,7 +129,7 @@ export function updateDeposit(event: Mint): void {
   //      - hourlyDepositUSD
   //      - cumulativeDepositUSD
   let marketHourly = getOrCreateMarketHourlySnapshot(event);
-  marketHourly.hourlyDepositUSD = marketDaily.dailyDepositUSD.plus(depositAmountUSD);
+  marketHourly.hourlyDepositUSD = marketHourly.hourlyDepositUSD.plus(depositAmountUSD);
```
Comment (this-username-is-taken) [inline subgraphs/inverse-finance/src/common/helpers.ts:132]: nice catch!

**Comment by this-username-is-taken** [comment]
Would you also update the versions in the README (Deployment Status section)?

**Comment by tnkrxyz** [comment]
I don't see any version info in the README (except for the methodogy version). Do you mean change: 
```
## Hosted Subgraph

- Hosted subgraph: https://thegraph.com/hosted-service/subgraph/tnkrxyz/inverse-finance
```

to 

```
## Hosted Subgraph

- Hosted subgraph: https://api.thegraph.com/subgraphs/name/messari/inverse-finance-ethereum
```
?

**Comment by this-username-is-taken** [comment]
Oh I meant updating this: https://github.com/messari/subgraphs/blob/master/README.md#development-status

**Comment by tnkrxyz** [comment]
> Oh I meant updating this: https://github.com/messari/subgraphs/blob/master/README.md#development-status

Ah, that makes sense. Updated.

The sync works and for the data points I checked using https://subgraph.messari.io they look good to me. The PR is ready for review.

**Comment by this-username-is-taken** [comment]
Looks good!

**Pull request merged**